packages/docusaurus-theme-classic/src/theme/Heading/index.tsx

```diff
@@ -24,7 +24,7 @@ export const MainHeading: HeadingComponent = function MainHeading({...props}) {
       <h1
         {...props}
         id={undefined} // h1 headings do not need an id because they don't appear in the TOC
-        className={styles.h1Heading}>
```

Comment:
Since we removed `h1Heading` class, should this change be marked as BC?

Reply:
I tend to prefer reporting breaking changes even if tiny, 

But I'd say CSS module classes do not need to be reported as breaking changes as it would only add noise.
It's somehow implicit it is internal code (similarly to the HTML tree) and the user probably understand this, as it's not easy to create a css module selector in the first place.